Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -1217,6 +1216,27 @@ public static InputUpdateType updateMask
             set { s_Manager.updateMask = value; }
         }
 
+        /// <summary>
+        /// Tells the Input System to ignore focus, and continue to trigger events and actions regardless of current focus state
+        /// </summary>
+        /// /// <remarks>
+        /// Off by default, this does not work on all platforms and devices, only those that can recieve their own input data while not in focus.
+        /// </remarks>
+        public static bool ignoreFocus
```

Comment:
Wondering if we should `#if` this one to make it clearer that this isn't a universal capability?

Reply:
Also wondering now whether it's clearer naming this explicitly after the run-in-background ability rather than relating it to focus. Maybe `receiveInputWhenInBackground` or `receiveInputWhenInBackgroundIfSupported`?

Reply:
I went with `runInBackground`.  Those property names are way too wordy.

Reply:
Names that leave no doubt about what they do in general are good. Wordy or not doesn't matter overly much. There's no tax on characters.